### DISCUSSION TITLE
Closed #5950 Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.

### DISCUSSION THREAD

**Discussion by Benedek012** [community/community#120381] [Discussions]
### Select Topic Area

Question

### Body

Good day.How long will the ban last? It was an accidental mistake, I couldn't fill it in correctly. 5954 problem,please

**Comment by LiteBrite82** [comment]
Thank you kindly for reaching out. If you are experiencing behavior against our Terms of Service or need to report GitHub account-related abuse, you can reach out to our private support team. We are unable to assist with or discuss personal account details here in Discussions.

Please visit this link for more details: https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam We hope that helps!
